### ISSUE TITLE
Apparent Python and Numpy_base compatibility

### ISSUE THREAD

**Issue by HamishGBrown** [scipion-em/scipion-em-emready#11]
Trying to install em-ready on a Linux Mint 20.1 workstation  and am getting the following error when the command `conda install -y -c conda-forge gfortran libxcrypt` is run as part of installation:

```
/home/bio21em1/anaconda3/envs/emready-1.0/lib/jvm/graalvm-189e927686-java17-22.3.0/languages/python/lib-python/3/importlib/_bootstrap.py:219: UserWarning: mkl-service package failed to import, therefore Intel(R) MKL initialization ensuring its correct out-of-the box operation under condition when Gnu OpenMP had already been loaded by Python process is not assured. Please install mkl-service package, see http://github.com/IntelPython/mkl-service
  return f(*args, **kwds)
Traceback (most recent call last):
  File "/home/bio21em1/anaconda3/envs/emready-1.0/lib/python3.8/site-packages/numpy/core/__init__.py", line 22, in <module>
    from . import multiarray
  File "/home/bio21em1/anaconda3/envs/emready-1.0/lib/python3.8/site-packages/numpy/core/multiarray.py", line 12, in <module>
    from . import overrides
  File "/home/bio21em1/anaconda3/envs/emready-1.0/lib/python3.8/site-packages/numpy/core/overrides.py", line 7, in <module>
    from numpy.core._multiarray_umath import (
ModuleNotFoundError: No module named 'numpy.core._multiarray_umath'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/bio21em1/anaconda3/envs/emready-1.0/bin/f2py", line 7, in <module>
    from numpy.f2py.f2py2e import main
  File "/home/bio21em1/anaconda3/envs/emready-1.0/lib/python3.8/site-packages/numpy/__init__.py", line 140, in <module>
    from . import core
  File "/home/bio21em1/anaconda3/envs/emready-1.0/lib/python3.8/site-packages/numpy/core/__init__.py", line 48, in <module>
    raise ImportError(msg)
ImportError: 

IMPORTANT: PLEASE READ THIS FOR ADVICE ON HOW TO SOLVE THIS ISSUE!

Importing the numpy C-extensions failed. This error can happen for
many reasons, often due to issues with your setup or how NumPy was
installed.

We have compiled some common reasons and troubleshooting tips at:

    https://numpy.org/devdocs/user/troubleshooting-importerror.html

Please note and check the following:

  * The Python version is: Python3.8 from "/home/bio21em1/anaconda3/envs/emready-1.0/bin/python"
  * The NumPy version is: "1.19.2"

and make sure that they are the versions you expect.
Please carefully study the documentation linked above for further help.

Original error was: No module named 'numpy.core._multiarray_umath'
```

The following stack exchange seems to suggest that this is a numpy and python version mismatch https://stackoverflow.com/questions/54665842/when-importing-tensorflow-i-get-the-following-error-no-module-named-numpy-cor

**Comment by azazellochg** [comment]
Thank you @HamishGBrown , are you using devel installation? We have updated devel to support emready-2.0 but the changes have not been released yet. I guess it's a good time to do it now.

**Comment by azazellochg** [comment]
@HamishGBrown plugin update (v3.1.0) has just been released, could you try and see if emready-2.0 works for you?

**Comment by azazellochg** [comment]
While working on this I found a different error with the installer. Now both production and devel should work fine with emready 2.0.
